pages/index.js

```diff
@@ -14,6 +14,29 @@ import includes from 'lodash/includes';
 import Link from 'next/link';
 
 export default class extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      // default to changelog if no forumlink is found
+      releaseNote: {
+        link: '/changelog'
+      }
+    }
+  }
+
+  componentDidMount() {
+    fetch(`https://forums.resin.io/c/etcher.json`)
+    .then((res) => res.json())
+    .then((releaseNotes) => {
+      return releaseNotes.topic_list.topics.find(topic => ( topic.title.includes(locals.version)))
```

Comment:
Looking at the whole [Etcher forum](https://forums.resin.io/c/etcher) it seems to be a pretty consistent pattern of "Etcher v _version_ release", so probably doesn't hurt to check for that, rather than just "_version_" ?